internal/policies/mount/mount_test.go

```diff
@@ -0,0 +1,106 @@
+package mount
+
+import (
+	"context"
+	"flag"
+	"os"
+	"os/user"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/ubuntu/adsys/internal/policies/entry"
+)
+
+func TestApplyPolicy(t *testing.T) {
+	t.Parallel()
+
+	entries := GetEntries(5, 0, 1, 0, []string{"\n"})
+
+	tests := map[string]struct {
+		entries    []entry.Entry
+		objectName string
+		computer   bool
+		uid        string
+		gid        string
+
+		wantErr bool
+	}{
+		"successfully generates mounts file":        {entries: entries},
+		"generates no file if there are no entries": {entries: []entry.Entry{}},
+
+		"error when user is not found":    {entries: entries, objectName: "dont exist", wantErr: true},
```

Comment:
With all the other tests, you can see that we comment them beforehand with `// Error cases`. Please follow the same convention.